instrumented/aboutUs/Info/Info.tsx

```diff
@@ -0,0 +1,32 @@
+import './info.scss';
+import olga from '../../assets/photo.jpg';
+import React from 'react';
```

Comment:
this react import should not be necessary

Reply:
There is an error when I try to remove React import